packages/marko-web-native-x/components/gtm-init.marko

```diff
@@ -0,0 +1,27 @@
+import defaultValue from "@parameter1/base-cms-marko-core/utils/default-value";
+import { get, getAsObject } from "@parameter1/base-cms-object-path";
+
+$ const { req } = out.global;
+$ const story = getAsObject(input, "story");
+$ const on = defaultValue(input.on, "load");
+$ const requestFrame = defaultValue(input.requestFrame, true);
+$ const targetTag = defaultValue(input.targetTag, "body");
+$ const push = {
```

Comment:
Now that i see that the push data needs the story, perhaps it would be better to move the push into its own track page view component. That way the user has a choice where to use it, and it keeps the init component cleaner